ext/phar/tests/fopen5.2.phpt

```diff
@@ -39,5 +38,5 @@ include $fname;
 --EXPECTF--
 Warning: fopen() expects at least 2 parameters, 0 given in %sfopen5.2.php on line %d
 hihi
-Warning: fopen(phar://%sfopen5.2.phar.php/notfound.txt): failed to open stream: phar error: "notfound.txt" is not a file in phar "%sfopen5.2.phar.php" in phar://%sfopen5.2.phar.php/index.php on line %d
-===DONE===
\ No newline at end of file
+Warning: fopen(notfound.txt): failed to open stream: No such file or directory in phar://%sfopen5.2.phar.php/index.php on line %d
+===DONE===
```

Comment:
This test should be deleted entirely, as we already have a non-5.2 version of this test.